controllers/admin/AdminImagesController.php

```diff
@@ -574,18 +620,32 @@ protected function _regenerateNewImages($dir, $type, $productsImages = false)
                             $image = str_replace('.', '_thumb.', $image);
                         }
 
+                        // todoimage: what if the original image is not jpg ?
```

Comment:
```suggestion
                        // todoimage: what if the original image is not jpg ?
```

I think it should be added as an issue or note in the EPIC, that's quite an important question :-)

Reply:
Original image has always an JPG extension no matter what, and format is detected automatically when generating thumbnail in ImageManager class.

We also found out that the fileType file serves no purpose, it stores the last format which was used to generate thumbnail, but it's not used anywhere.

Reply:
ok, so if it's answered already it could be deleted :D

Reply:
yes it's something I didn't understand at first, commited it accidently :D